src/components/progressBar/actionSlider.js

```diff
@@ -1,26 +1,93 @@
 import React from 'react';
+import cx from 'classnames';
+
+import Icon from '../Icon';
 
 const CLASS_NAMES = {
+    SLIDER_WRAPPER: 'slider-wrapper',
     SLIDER_CONTAINER: 'slider-container',
     SLIDER_HEADER: 'slider-header',
     SLIDER_TITLE: 'slider-title',
-    SLIDER_TEXT: 'slider-text'
+    SLIDER_TEXT: 'slider-text',
+    SLIDER_ICONS_WRAPPER: 'slider-icons-wrapper',
+    SLIDER_LEFT_ARROW: 'slider-left-arrow',
+    SLIDER_RIGHT_ARROW: 'slider-right-arrow',
+    SLIDING_ELEMENT: 'sliding-element'
+}
+
+const sliderData = [
```

Comment:
Assuming this is prototype / demo?

Reply:
yes, its only for the demo purposes, needs refactor